application/models/services/QuestionAggregateService.php

```diff
@@ -143,7 +143,9 @@ public function delete($surveyId, $questionId)
      */
     public function deleteMany($surveyId, $questionIds)
     {
+        $survey = \Survey::model()->findByPk($surveyId);
```

Comment:
You need to use dependency injection to get the model instance.

Reply:
fixed